spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -1025,6 +1034,11 @@ function timeoutPacket(
       provableStore.set(nextSequenceAckPath(packet.srcPort, packet.srcChannel), nextSequenceAck)
     }
 
+    // if there are no in-flight packets on our end, we can automatically go to FLUSHCOMPLETE
+    if channel.flushStatus === FLUSHING && pendingInflightPackets(portIdentifier, channelIdentifier) == nil {
+        channel.flushState = FLUSHCOMPLETE
+    }
```

Comment:
I would move this to be next to packet deletion (as the logic is related). In addition, when ORDERED channels close, they should set flushing to `NOTINFLUSH` so that the counterparty can safely close as well. I think if the flushing is in `FLUSHING` or `FLUSHCOMPLETE` then the counterparty channel close will fail on proof validation (since it expects the counterparty to be in `NOTINFLUSH`